src/entities/Staff.java

```diff
@@ -11,46 +17,47 @@ public class Staff implements Client, Serializable {
     private Cinema cinema;   //associated cinema of the staff. Where the staff works
 
     public Staff(String username, String password, Cinema cinema, ClientController c, MovieController mvc) {
```

Comment:
Since mvc and c is removed from the attribute of staff. How does the methods in this class going to use mvc and c methods?

Reply:
I think the only way is probably how you did it by importing controllers.ClientController etc.